hr_phone_validation/tests/test_phone_validation.py

```diff
@@ -0,0 +1,12 @@
+from odoo.tests.common import TransactionCase
+
+
+class TestHrPhoneValidation(TransactionCase):
+
+    def test_phone_validation(self):
+        Employee = self.env['hr.employee']
```

Comment:
Variable name should be lowercase as `hr_employee`: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#variable-names

Reply:
In this case, you should change variable `Employee` into `hr_employee_env` to indicate it's environment variable.